### PR TITLE
feat(grid-layout): implement grid playground

### PR DESCRIPTION
**Pull request by JoCa96**
Implement grid playground to show how the grid system behaves and works.
Is only being intended to be used through the full-size storybook component example page.

### CODE REVIEW

**Review by MajaZarkova — APPROVED** [review]

File: packages/sit-onyx/src/components/GridPlayground/GridPlayground.stories.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import type { Meta, StoryObj } from "@storybook/vue3";
+import Grid from "./GridPlayground.vue";
+
+/**
+ * The GridPlayground allows to test and play around with the Onyx grid system
+ */
+const meta: Meta<typeof Grid> = {
+  title: "utility/GridPlayground",
```
Comment (BoppLi) [inline packages/sit-onyx/src/components/GridPlayground/GridPlayground.stories.ts:8]: this is a test

File: packages/sit-onyx/src/components/GridPlayground/GridPlayground.stories.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  component: Grid,
+};
+
+export default meta;
+type Story = StoryObj<typeof Grid>;
+
+/**
```
Comment (BoppLi) [inline packages/sit-onyx/src/components/GridPlayground/GridPlayground.stories.ts:15]: adsfdsf

**Review by BoppLi — DISMISSED** [review]
2/4 done, this is a demo for maja

File: packages/sit-onyx/src/components/GridPlayground/GridPlayground.stories.ts (hunk 1 above)
Comment (BoppLi) [reply packages/sit-onyx/src/components/GridPlayground/GridPlayground.stories.ts:8]: no we won't change it.

**Pull request merged**